charts/lagoon-logging/values.yaml

```diff
@@ -0,0 +1,91 @@
+# Default values for lagoon-logging.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+logsDispatcher:
+
+  name: logs-dispatcher
+
+  replicaCount: 2
+
+  image:
+    repository: amazeeiolagoon/logs-dispatcher
+    pullPolicy: Always
+    # Overrides the image tag whose default is the chart version.
+    tag: logs-overhaul-v2
```

Comment:
I'll add a follow-up PR to change this to `master` once merged, and then to the latest release version once `logs-dispatcher` has made it into a release.

Reply:
done